apps/coscrad-frontend/src/components/resources/terms/term-detail.presenter.tsx

```diff
@@ -5,4 +5,9 @@ import { GenericDetailPresenter } from '../../../utils/generic-components/presen
 // TODO[https://www.pivotaltracker.com/story/show/183681722] expose commands
 export const TermDetailPresenter = (
     dataAndActions: IDetailQueryResult<ITermViewModel>
-): JSX.Element => <GenericDetailPresenter {...dataAndActions} />;
+): JSX.Element => (
```

Comment:
I might need to reset my changes to this file so not to step on Justin's toe with his work.

Reply:
done- double check this though